app/controllers/api/v2/common_parameters_controller.rb

```diff
@@ -19,9 +19,18 @@ def show
 
       def_param_group :common_parameter do
         param :common_parameter, Hash, :required => true, :action_aware => true do
-          param :name, String, :required => true
-          param :value, String, :required => true
```

Comment:
Yes. I first though to keep it as value and change it in `rename_common_parameters` but since I'm adding a lot more to the api and the result will have `default_value` I thought it's better to change it.
I can keep it as value and only present it as default_value (it's a little confusing but might be better then breaking the api).